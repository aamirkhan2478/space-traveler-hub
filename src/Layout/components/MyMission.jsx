import React from "react";

const MyMission = () => {
  return (
    <>
      <h3>My Mission</h3>
    </>
  );
};

export default MyMission;
