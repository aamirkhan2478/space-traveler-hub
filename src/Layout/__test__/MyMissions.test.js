import renderer from 'react-test-renderer';
import MyMission from './__mock__/MyMissions';

it('renders missions correctly', () => {
  const tree = renderer.create(<MyMission />).toJSON();
  expect(tree).toMatchSnapshot();
});
