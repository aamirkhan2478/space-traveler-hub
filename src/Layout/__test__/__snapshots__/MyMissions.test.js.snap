// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders missions correctly 1`] = `
Array [
  <h3>
    My Mission
  </h3>,
  <div
    className="list-group"
    onKeyDown={[Function]}
  >
    <div
      className="list-group-item"
      data-rr-ui-event-key={null}
      onClick={[Function]}
    >
      Thaicom
    </div>
  </div>,
]
`;
